pkg/apis/pingcap/v1alpha1/backup.go

```diff
@@ -118,6 +118,18 @@ func IsBackupScheduled(backup *Backup) bool {
 	return condition != nil && condition.Status == corev1.ConditionTrue
 }
 
+// IsBackupRunning returns true if a Backup is Running.
+func IsBackupRunning(backup *Backup) bool {
+	_, condition := GetBackupCondition(&backup.Status, BackupRunning)
+	return condition != nil && condition.Status == corev1.ConditionTrue
+}
+
+// IsBackupPrepare returns true if a Backup is Prepare.
+func IsBackupPrepare(backup *Backup) bool {
```

Comment:
```suggestion
// IsBackupPrepared returns true if a Backup is Prepare.
func IsBackupPrepared(backup *Backup) bool {
```

Reply:
it's a little weird because we already defined a stage named `Prepare` after `Running`.

Reply:
Yes for the stage name.
But I think the function name need not be identical to the stage name. The function name should be more easy to understand.

Reply:
renamed in https://github.com/pingcap/tidb-operator/pull/3696/commits/bdf60cc4106815e9b19f4204ba56b67ffaf5a2ab.